pkgs/development/libraries/amdvlk/default.nix

```
@@ -0,0 +1,88 @@
+{ stdenv
+, lib
+, fetchpatch
+, fetchRepoProject
+, cmake
+, ninja
+, patchelf
+, perl
+, pkgconfig
+, python3
+, expat
+, libdrm
+, ncurses
+, openssl
+, wayland
+, xorg
+, zlib
+}:
+
+with lib;
+
+stdenv.mkDerivation rec {
+  pname = "amdvlk";
+  version = "2020.Q2.5";
+
+  src = fetchRepoProject {
+    name = "${pname}-src";
+    manifest = "https://github.com/GPUOpen-Drivers/AMDVLK.git";
+    rev = "refs/tags/v-${version}";
+    sha256 = "008adby8vx12ma155x64n7aj9vp9ygqgij3mm3q20i187db7d1ab";
+  };
+
+  buildInputs = [
+    expat
+    ncurses
+    openssl
+    wayland
+    xorg.libX11
+    xorg.libxcb
+    xorg.xcbproto
+    xorg.libXext
+    xorg.libXrandr
+    xorg.libXft
+    xorg.libxshmfence
+    zlib
+  ];
+
+  nativeBuildInputs = [
+    cmake
+    ninja
+    patchelf
+    perl
+    pkgconfig
+    python3
+  ];
+
+  rpath = lib.makeLibraryPath [
+    libdrm
+    stdenv.cc.cc.lib
+    xorg.libX11
+    xorg.libxcb
+    xorg.libxshmfence
+  ];
+
+  cmakeDir = "../drivers/xgl";
+
+  installPhase = ''
+    mkdir -p $out/lib $out/share/vulkan/icd.d
+
+    cp icd/amdvlk64.so $out/lib/
+    cp ../drivers/AMDVLK/json/Redhat/amd_icd64.json $out/share/vulkan/icd.d/
+    substituteInPlace $out/share/vulkan/icd.d/amd_icd64.json --replace \
+      '/usr/lib64' "$out/lib"
```

```suggestion

    substituteInPlace $out/share/vulkan/icd.d/amd_icd64.json --replace \
      "/usr/lib64" "$out/lib"
```

Double quotes for consistency.